File: mRemoteV1/Connection/Connection.Protocol.SSH1.cs


namespace mRemoteNG.Connection.Protocol
{
	public class SSH1 : PuttyBase
	{
				
		public SSH1()
		{
			this.PuttyProtocol = Putty_Protocol.ssh;
			this.PuttySSHVersion = Putty_SSHVersion.ssh1;
		}
				
		public enum Defaults
		{
			Port = 22
		}
	}
}

File: mRemoteV1/Connection/Connection.Protocol.SSH1.vb
Namespace Connection

    Namespace Protocol
        Public Class SSH1
            Inherits PuttyBase

            Public Sub New()
                Me.PuttyProtocol = Putty_Protocol.ssh
                Me.PuttySSHVersion = Putty_SSHVersion.ssh1
            End Sub

            Public Enum Defaults
                None = 0
                Port = 22
            End Enum
        End Class
    End Namespace

End Namespace